migrator/README.md

```diff
@@ -18,17 +19,17 @@ A migration can read from any of the databases, make changes to the data or to t
 
    All migrations were of the form `m_{currentDBVersion}_to_m_{currentDBVersion+1}_{summary_of_migrations}` .
 
-2. Release 3.73 brought the database `Postgres` as Technical preview. This had the consequence that 
+2. Release 3.73 and 3.74 brought the database `Postgres` as Technical preview. This had the consequence that
 `Postgres` was a new potential datastore targeted by the migrator.
 
-   In 3.73, two sets of data migrations were possible: key-value store migrations, and data move migrations
+   In these versions, two sets of data migrations were possible: key-value store migrations, and data move migrations
```

Comment:
Technically, key-value store migrations are not allowed anymore after 3.73.
With 3.74, there are already postgres -> postgres migrations.
I'm not sure how to formulate things clearly here, though.